### ISSUE TITLE
Failed to execute 'importScripts'

### ISSUE THREAD

**Issue by abdel-abdelrazek** [odoe/jsapi-react-template#3]
Hi I received this error as soon as i run the project.
![screenshot from 2019-02-08 06-37-20](https://user-images.githubusercontent.com/38641993/52476285-2ba1d800-2b6c-11e9-9906-654e5d50b450.png)

**Comment by odoe** [comment]
This is incredibly intermittent, sometimes it works, sometimes not. Yarn install seems to be the most stable, not sure if it's an issue with the way webpack does the build or something in the API. Will continue to investigate.

**Comment by tomwayson** [comment]
Yep, me too. I _thought_ I already opened an issue for this... but turns out it's only in an unsent email in my drafts folder.

![image](https://user-images.githubusercontent.com/662944/52604409-9e86a980-2e1f-11e9-99d7-96934cb20b94.png)

**Comment by tomwayson** [comment]
This did seem intermittent at first to me, but since I've pulled master (w/ hooks) it seems to be happening more often than not.

I can't yet reproduce it consistently, but I have noticed that once I've seen that error, I will continue to see it until whenever I reload the page (i.e. manually or via live reload) until I restart the dev server, which seems to have a 50/50 chance of starting up w/o that error.

**Comment by tomwayson** [comment]
FYI, sadly, #4 does **not** fix this.

Not surprising, this seems like a service worker thing.

**Comment by odoe** [comment]
Which is odd since service workers should be disabled in dev mode.

**Comment by odoe** [comment]
This doesn't happen in another react app I am working on and does not happen in the jsapi-resources webpack demo app. I think I'll need to start this one from scratch. Maybe it's a dependency somewhere.

**Comment by odoe** [comment]
This has been updated and I'm no longer seeing this issue. If someone else could verify, that would be great.

**Comment by davewilton** [comment]
@odoe I've run it 10 times and it appears to be working for me. I'd normally get the issue more often than that. I tried in my old project and it failed first time.

Any pointers on what the fix was so I can sort my other project?

Cheers

**Comment by odoe** [comment]
I wish I had a better idea. It could be the dojo webpack plugin updates. Try to update all deps using `npm outdated` as a guide
